backend/api/elite2Api.js

```diff
@@ -25,10 +25,8 @@ const elite2ApiFactory = client => {
       `api/case-notes/usage?type=KA&subType=KS&staffId=${staffId}&numMonths=1&${encodeOffenderNumbers(offenderNumbers)}`
     )
   const csraList = (context, offenderNumbers) => post(context, 'api/offender-assessments/csra/list', offenderNumbers)
-  const userCaseLoads = context => get(context, 'api/users/me/caseLoads')
-  const currentUser = context => get(context, 'api/users/me')
-  const userLocations = context => get(context, 'api/users/me/locations')
-  const getUserAccessRoles = context => get(context, 'api/users/me/roles')
+  const userCaseLoads = context => (context.authSource !== 'auth' ? get(context, 'api/users/me/caseLoads') : [])
```

Comment:
You could add a function to contextProperties.js to wrap the context.authSource !== 'auth' bit.